src/index.html

```diff
@@ -224,45 +224,53 @@ <h2>We rely on individual donors to fund our work</h2>
+    <p><a href="tel:+14154296753">+1-415-429-6753</a></p>
     
     <nav class="social-menu">
         <ul>
             <!-- <li><a class="icon-replace fa-instagram" href="#">Instagram</a></li> -->
-            <li><a class="icon-replace fa-twitter" href="#">Twitter</a></li>
-            <li><a class="icon-replace fa-mastodon" href="#">Mastodon</a></li>
-            <li><a class="icon-replace fa-facebook" href="#">Facebook</a></li>
-            <li><a class="icon-replace fa-linkedin" href="#">LinkedIn</a></li>
+            <li><a class="icon-replace fa-twitter" href="https://twitter.com/creativecommons" target="_blank">Twitter</a></li>
+            <li><a class="icon-replace fa-mastodon" href="https://mastodon.social/@creativecommons" target="_blank">Mastodon</a></li>
+            <li><a class="icon-replace fa-facebook" href="https://www.facebook.com/creativecommons" target="_blank">Facebook</a></li>
+            <li><a class="icon-replace fa-linkedin" href="https://www.linkedin.com/company/creative-commons/" target="_blank">LinkedIn</a></li>
         </ul>
     </nav>
     </div>
     
     <div class="subscribe">
     <h2>Subscribe to our Newsletter</h2>
+    <form action="https://creativecommons.us4.list-manage.com/subscribe/post?u=fd30364b6577b471373d6076c&amp;id=4603fe102a" method="post" id="mc-embedded-subscribe-form" name="mc-embedded-subscribe-form" class="validate newsletter" target="_blank" novalidate="">
+        <input type="email" value="" name="EMAIL" class="email input" id="mce-EMAIL" placeholder="Your email" required="">
+        <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+        <div style="position: absolute; left: -5000px" aria-hidden="true">
+          <input type="text" name="b_fd30364b6577b471373d6076c_4603fe102a" tabindex="-1" value="">
+        </div>
+        <input type="submit" value="subscribe" id="mc-embedded-subscribe" class="button small">
+      </form>
     </div>
     
     <div class="donate">
         <h2>Support Our Work</h2>
         <p>Our work relies on you! Help us keep the Internet free and open.</p>
-        <a class="donate icon-attach cc-heart-filled">Donate Now</a>       
+        <a class="donate icon-attach cc-heart-filled" href="https://www.classy.org/give/313412/#!/donation/checkout?c_src=website&c_src2=top-of-page-banner" target="_blank">Donate Now</a>       
     </div>
     
 
     <div class="license">
-        <p>Except where otherwise <a href="#">noted</a>, content on this site is licensed under a <a href="#">Creative Commons Attribution 4.0 International license</a>. Icons by <a href="#">Font Awesome</a>.</p>
+        <p>Except where otherwise <a href="/policies/#license">noted</a>, content on this site is licensed under a <a href="https://creativecommons.org/licenses/by/4.0/">Creative Commons Attribution 4.0 International license</a>. Icons by <a href="https://fontawesome.com/" target="_blank">Font Awesome</a>.</p>
```

Comment:
I'm fine with that and a big supporter of relative URLs as much a possible.

But, I went back and forth on this one a lot.

My reasoning here was the more general areas of the footer/header that are not localized (like the primary nav and footer) should be absolute links so no matter where the footer/header is implemented it has the correct URLs and only the primary/footer nav need be setup for that particular implementation specifically with relative URLs. 

That way it was easier to copy/paste the two "components" into any site from this baseline model, and then only the localized nav need be implemented manually or programmatically in whatever way made sense.

And secondarily the license links are linked all over the internet with absolute URLs back to these pages, so I thought those two points tipped in that direction, but I'm fine with changing it and us just knowing to modify it when we put it other places too. 

The counter-argument I kept having with myself was that the above absolute/relative rule split would make sense when we migrate it back to vocabulary, but since this is the prototype for creativecommons.org we should go ahead and "localize it" throughout, and kept going back and forth. 

Whichever you think will work best overall, I'll swap it, and we can always mess with it later, lol.

Reply:
That all makes sense, thank you for the response and thinking through it. It helps!

I'll localize here with path-only URLs where possible. I _should_ have, but didn't think about the accuracy and flexibility of the dev/prod division we'd want to support for building and testing and that definitely tips my internal arguments over into precisely where you are. I'll go back through and make sure I've set path-only/localized relative URLs here and in the forthcoming header PR too so it's resolved :)

Thank you! ❤️